CMSgov/mint-app
# [EASI-4441] Cypress change history

# EASI-4441


## Changes and Description

- Updated Read-view link for non-assessment users (previously was only showing for assessment)
- Added cypress test
  - Tested search, sort, and collapsible link



## How to test this change

Verify cypress tests run successfully
Verify read view CH link appears for non assessment users

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
